src/components/header.js

```diff
@@ -21,12 +21,15 @@ function Header() {
     if (path !== '/')
       return (
         <OrangeFCCLogo
-          className="logo"
-          alt={`${site.siteMetadata.title} Logo`}
+          className="logo logoOrange"
+          alt={`${site.siteMetadata.title} Orange Logo`}
         />
       );
     return (
-      <BlueFCCLogo className="logo" alt={`${site.siteMetadata.title} Logo`} />
+      <BlueFCCLogo
+        className="logo logoBlue"
```

Comment:
What do you think about "logoBlue" instead of "logo logoBlue" to reduce typing?

Reply:
I'll disagree; the `logo` class has rules that apply to both, while `logoBlue` and `logoOrange` relate only to their color. This seems like a proper separation of concerns.